.github/workflows/arbitrator-ci.yml

```diff
@@ -50,7 +50,7 @@ jobs:
         id: install-rust
         with:
           profile: minimal
-          toolchain: "1.66.1"
+          toolchain: "1.67.1"
```

Comment:
Let's change this and the others to `stable`

Reply:
I had it this way because if we pin the version we don't get it surprise breaking when rust updates and instead can update rust at our leisure. But happy to have it be stable instead.

Reply:
Let's keep with the latest Rust so that CI changes when there's new Rust releases :)